lib/europeana/blacklight/api_repository.rb

```diff
@@ -36,9 +36,11 @@ def search(params = {})
       # @param id [String] Europeana record ID, with leading slash
       # @return [Hash] Record's hierarchy data, or false if it has none
       def fetch_document_hierarchy(id, relation = nil, options = {})
-        connection::Record.new(id).hierarchy.tap do |hierarchy|
-          relation.nil? ? hierarchy.with_family : hierarchy.send(relation, options)
-        end
+        fail ArgumentError, "Invalid relation \"#{relation}\"" unless relation.nil? ||
+          %w(parent children following_siblings preceding_siblings).include?(relation)
```

Comment:
Use 4 (not 2) spaces for indenting a condition in an `unless` statement spanning multiple lines.